Userland/Services/WindowServer/ConnectionFromClient.cpp

```diff
@@ -434,7 +434,7 @@ Messages::WindowServer::SetWindowRectResponse ConnectionFromClient::set_window_r
 {
     auto it = m_windows.find(window_id);
     if (it == m_windows.end()) {
-        did_misbehave("SetWindowRect: Bad window ID");
+        did_misbehave(String::formatted("SetWindowRect: Bad window ID: {}", window_id).characters());
```

Comment:
This looks like a change that is not related to setting window rects.

Reply:
right, that's a debugging leftover...